### ISSUE TITLE
why the list include external javascript file?

### ISSUE THREAD

**Issue by ghost** [reek/anti-adblock-killer#3730]
what's the use of the line ?

###k2Uw7isHrMm5JXP1Vwdxc567ZKc1aZ4I
||reek.github.io/anti-adblock-killer/k2Uw7isHrMm5JXP1Vwdxc567ZKc1aZ4I.js|$script

no answers have been given on

https://github.com/reek/anti-adblock-killer/issues/2147
https://github.com/reek/anti-adblock-killer/issues/688

thanks you

**Comment by uBlock-user** [comment]
yes it has, read again - https://github.com/reek/anti-adblock-killer/issues/2147#issuecomment-261722808

**Comment by uBlock-user** [comment]
Also this project is dead, about time you remove AAK.

**Comment by ghost** [comment]
no it have not been answered, i removed it anyways i will never run such thing in my browser

**Comment by uBlock-user** [comment]
Good for you, should have removed it a long time ago then.

**Comment by jspenguin2017** [comment]
It's used to check if AAK-List is installed, I don't do that in uBR nor AAK-Cont, but I plan on doing it for ABR2, I probably will have it run once a day with XHR, probably part of libmicro.

**Comment by jspenguin2017** [comment]
Hum, there are a few left-over internal filters.
